rust/openvasd-tests/entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -e
+
+OPENVASD_PATH=/opt/openvasd_target/debug/openvasd
+
+redis-server --daemonize yes --bind 127.0.0.1 --port 6379
+
+until redis-cli ping > /dev/null 2>&1; do
```

Comment:
if you change to unix socket the command would change to redis-cli -S /run/redis/redis.sock ping, just a note if you change that.